docs/pages/blog/mui-x-v7-beta.md

```diff
@@ -0,0 +1,247 @@
+---
+title: MUI X v7 is now in beta
+description: Check out what's new and what's next for v7 stable.
+date: 2024-01-31T00:00:00.000Z
+authors: ['josefreitas']
+tags: ['MUI X', 'News']
+card: true
+---
+
+<div style="max-width: 692px; width: 100%; height: 230px; overflow: hidden; margin-bottom: 16px;">
+  <a href="https://github.com/mui/mui-x/releases/tag/v7.0.0-beta.0">
+    <img src="/static/blog/mui-x-v7-beta/intro.png" alt="MUI X v7 beta release" width="1200" height="400" style="width: 100%; height: 100%; object-fit: cover; object-position: center;" />
+  </a>
+</div>
+
+It's the end of January, and we're ready to start rolling out some of the plans for 2024!
+We have new components, exciting features, and a lot of improvements for both developers and end-users.
+
+Starting [now](https://github.com/mui/mui-x/releases/tag/v7.0.0-beta.0), MUI X v7 is in beta.
+Most breaking changes are in place, and we shift our focus towards refining and expanding over the new version.
+
+## Table of contents
+
+- [What's New](#whats-new)
+  - [Data Grid](#data-grid)
+    - [Sticky Headers](#sticky-headers)
+    - [Server-Side Data Source](#server-side-data-source)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+    - [Date Object Support in Filter Model](#date-object-support-in-filter-model)
+    - [New Stable Features](#new-stable-features)
+  - [TreeView](#treeview)
+    - [RichTreeView](#richtreeview-new-component)
+  - [Charts](#charts)
+    - [Reference Line](#reference-line)
+  - [Date Pickers](#date-pickers)
+    - [Date Time Range Picker](#date-time-range-picker)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+    - [Support for date-fns v3](#support-for-date-fns-v3)
+- [What's Next](#whats-next)
+- [Migration Guide](#migration-guide)
+- [How to Get Involved](#how-to-get-involved)
+
+# What's new
+
+## Data Grid
+
+We have introduced critical breaking changes designed not only to enhance the developer experience but also to bring the user experience to a new level.
+
+### Sticky Headers
+
+The column headers and pinning implementations have been refactored for a significantly better user experience.
+You'll notice a more responsive Data Grid with smoother scrolling and better screen reader support.
+The video below showcases some of these improvements.
+
+<video preload="metadata" style="margin-bottom: 10px;" autoplay muted loop playsinline width="690" height="417">
```

Comment:
Maybe to add the controls, I wanted to play with the video at one point but couldn't. It's longer than 10s so could be better:

```suggestion
<video preload="metadata" style="margin-bottom: 10px;" autoplay muted loop playsinline width="690" height="417" controls>
```